common-lib/common/panels/requests/stat/errors.libsonnet

```diff
@@ -0,0 +1,17 @@
+local g = import '../../../g.libsonnet';
+local generic = import '../../generic/stat/main.libsonnet';
+local base = import './base.libsonnet';
+local stat = g.panel.stat;
+// Uptime panel. expects duration in seconds as input
```

Comment:
I think it'd be beneficial to extract these requirements to a readme file within the common lib to have a central place around expectations and maybe some screenshots like you included in the description body.

Thoughts?

Reply:
Sorry, this comment was unrelated. Dropped it.

Reply:
As of images, I'll try to provide some next time.